.github/workflows/github-token-rotation.yml

```
@@ -0,0 +1,47 @@
+      contents: read
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/sandbox-test-codepipeline-role
+          aws-region: eu-central-1
+
+      - name: Rotate GitHub Token
+        env:
+          SECRET_NAME: github-token
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:16:44: Double quote to prevent globbing and word splitting

(shellcheck)